cmd/create.go

```diff
@@ -146,7 +147,14 @@ func runCreate(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
-	provisioner, err := getProvisioner(provider, accessToken, secretKey, organisationID, region)
+	var subscriptionID string
+	var accessTokenFile string
+	if provider == "azure" {
+		subscriptionID, _ = cmd.Flags().GetString("subscription-id")
+		accessTokenFile, _ = cmd.Flags().GetString("access-token-file")
```

Comment:
What if the value is being passed via `--access-token $(cat file.txt)`?

Reply:
Simply passing in the file contents won't work, because Azure Go SDK expects a file.

A possible way is to use `auth.NewAuthorizerFromEnvironment()`. This way we can allow the user uses 3 parameters and then we assign to the corresponding environment variables below:
```
    - `AZURE_TENANT_ID`: Specifies the Tenant to which to authenticate.
    - `AZURE_CLIENT_ID`: Specifies the app client ID to use.
    - `AZURE_CLIENT_SECRET`: Specifies the app secret to use.
```

Above 3 parameters can be found in the access token file, here is an example:
```
{
  "clientId": "<uuid>"
  "clientSecret": "<uuid>",
  "subscriptionId": "<uuid>",
  "tenantId": "<uuid>",
  "activeDirectoryEndpointUrl": "https://login.microsoftonline.com",
  "resourceManagerEndpointUrl": "https://management.azure.com/",
  "activeDirectoryGraphResourceId": "https://graph.windows.net/",
  "sqlManagementEndpointUrl": "https://management.core.windows.net:8443/",
  "galleryEndpointUrl": "https://gallery.azure.com/",
  "managementEndpointUrl": "https://management.core.windows.net/"
}
```

We may also need to handle the endpoint URL. Azure has different endpoints/base URL: global endpoint as showed above, US Gov, China, etc.

More details:
https://github.com/Azure/azure-sdk-for-go#more-authentication-details